### PR TITLE
Fifteen actions. Includes 2013 and recent.

### PR DESCRIPTION
**Pull request by nataliyaned**

### CODE REVIEW

File: actions.csv [hunk 1]
```diff
@@ -1,160 +1,175 @@
...
-104,2019/04/04,https://www.wired.com/story/microsoft-employees-protest-treatment-women-ceo-nadella/,protest,discrimination,"A group of Microsoft employees appeared at an employee meeting with CEO Satya Nadella Thursday to protest the companies's treatment of women. It's not clear how many were part of the protest, but some female and male employees at the event wore all white, inspired by the congresswomen who wore suffragette white to the State of the Union in February.","seattle, washington, usa",microsoft,None,"white_collar_workers, sexism",organizejs
-105,2019/04/10,"https://www.recode.net/2019/4/10/18304877/amazon-climate-change-employees-tech-activism, https://www.vox.com/business-and-finance/2019/4/11/18306389/amazon-employees-climate-change-plan",open_l
```
Comment (organizejs) [inline actions.csv]: foxxcon in brazil.. thats really interesting, had no idea.

Comment (nataliyaned) [reply actions.csv]: Yeah, quite unexpected!

**Comment by nataliyaned** [comment]
The historical cases are referenced in articles talking about recent events. Very often when I find one, I end up hearing about three more. There's a chain effect. That's what happened with FoxConn this time around and Amazon last time.

File: actions.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (nataliyaned) [reply actions.csv]: I was on the fence about this one too. We did include the OkCupid website protest against Mozilla so I went off of that precedent.

[the remark this answers is not in the document]
Comment (organizejs) [reply actions.csv]: ok makes sense

**Pull request merged**